src/status_im/chat/events.cljs

```diff
@@ -134,15 +134,19 @@
 
 (defn- navigate-to-chat
   "Takes coeffects map and chat-id, returns effects necessary for navigation and preloading data"
-  [chat-id {:keys [navigation-replace?]} {:keys [db] :as cofx}]
-  (if navigation-replace?
+  [chat-id {:keys [modal? navigation-replace?]} {:keys [db] :as cofx}]
+  (if modal?
     (handlers-macro/merge-fx cofx
-                             (navigation/replace-view :chat)
+                             {:dispatch [:navigate-to-modal :chat-modal]}
```

Comment:
Changes related to navigation will cause conflicts with https://github.com/status-im/status-react/pull/5606